packages/apputils/src/dialog.tsx

```diff
@@ -249,7 +249,9 @@ export class Dialog<T> extends Widget {
         this._primary = el as HTMLElement;
       }
     }
-    this._primary.focus();
+    if (this._primary) {
+      this._primary.focus();
+    }
```

Comment:
We can take advantage of the nullish operator

```suggestion
    this._primary?.focus();
```

FYI, in JavaScript this will be translated to the condition you have explicitly written.